tests/lib/rule-tester/rule-tester.js

```diff
@@ -97,6 +97,267 @@ describe("RuleTester", () => {
         ruleTester = new RuleTester();
     });
 
+    describe("only", () => {
+        describe("`itOnly` accessor", () => {
+            describe("when `itOnly` is set", () => {
+                before(() => {
+                    RuleTester.itOnly = sinon.spy();
+                });
+                after(() => {
+                    RuleTester.itOnly = void 0;
+                });
+                beforeEach(() => {
+                    RuleTester.itOnly.resetHistory();
+                    ruleTester = new RuleTester();
+                });
+
+                it("is called by exclusive tests", () => {
+                    ruleTester.run("no-var", require("../../fixtures/testers/rule-tester/no-var"), {
+                        valid: [{
+                            code: "const notVar = 42;",
+                            only: true
+                        }],
+                        invalid: []
+                    });
+
+                    sinon.assert.calledWith(RuleTester.itOnly, "const notVar = 42;");
+                });
+            });
+
+            describe("when `it` is set and has an `only()` method", () => {
+                before(() => {
+                    RuleTester.it.only = () => {};
+                    sinon.spy(RuleTester.it, "only");
+                });
+                after(() => {
+                    RuleTester.it.only = void 0;
+                });
+                beforeEach(() => {
+                    RuleTester.it.only.resetHistory();
+                    ruleTester = new RuleTester();
+                });
+
+                it("is called by tests with `only` set", () => {
+                    ruleTester.run("no-var", require("../../fixtures/testers/rule-tester/no-var"), {
+                        valid: [{
+                            code: "const notVar = 42;",
+                            only: true
+                        }],
+                        invalid: []
+                    });
+
+                    sinon.assert.calledWith(RuleTester.it.only, "const notVar = 42;");
+                });
+            });
+
+            describe("when global `it` is a function that has an `only()` method", () => {
+                let originalGlobatItOnly;
+
+                before(() => {
+
+                    /*
+                     * We run tests with `--forbid-only`, so we have to override
+                     * `it.only` to prevent the real one from being called.
+                     */
+                    originalGlobatItOnly = it.only;
+                    it.only = () => {};
+                    sinon.spy(it, "only");
+                });
+                after(() => {
+                    it.only = originalGlobatItOnly;
```

Comment:
```suggestion
                let originalGlobalItOnly;

                before(() => {

                    /*
                     * We run tests with `--forbid-only`, so we have to override
                     * `it.only` to prevent the real one from being called.
                     */
                    originalGlobalItOnly = it.only;
                    it.only = () => {};
                    sinon.spy(it, "only");
                });
                after(() => {
                    it.only = originalGlobalItOnly;
```

just a small typo